papers/drivendata/main.md

```diff
@@ -434,6 +450,10 @@ Complementary ECDF of F1 score for label classes in the holdout set with at leas
 Precision and recall values for a selection of label classes. The top half are the top 10 label classes ranked by F1 score, while the bottom half are the bottom 10 label classes ranked by F1 score. The parenthetical annotation gives the number of observations of that class.
 :::
 
+For an indirect comparison[^footnote-image-benchmarking], SpeciesNet, the model used by Wildlife Insights, report a 81.9% accuracy, 80.7 species-weighted-average F1 score, and 54.2 macro-averaged F1 score[@doi:10.1049/cvi2.12318]. This shows that Zamba's pretrained model is in a rough comparable range of performance as other published results.
```

Comment:
would be nice to include the number of labels in SpeciesNet or the eval dataset  (if different) since that's important context for interpreting f1

> the model used by Wildlife Insights

is "used" correct here? did they develop it?